core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -19,7 +19,7 @@
     </continue-button>
   </div>
 
-  <div>
+  <div class="interaction-container">
```

Comment:
Make sure to prefix this with conversation-skin-..., like all other classes in this file.

Reply:
Done.